src/main/jte/bookings.jte

```diff
@@ -45,6 +46,11 @@
                                 <td>${booking.getEndDate().toString()}</td>
                                 <td>${booking.getShareRoomType()}</td>
                                 <td>${booking.getPriority()}</td>
+                                <td>
+
+                                    <a href="/bookings/delete/${booking.getId()}" class="btn btn-ghost">
```

Comment:
This should NOT be a GET request, as those may, for example, be preloaded by browsers.
Instead, the table could be wrapped in a form to create a DELETE (or similar) request.
See [`tinyib`](https://code.rocket9labs.com/tslocum/tinyib) for an example of this (I'll show you an example of this in practice tomorrow)